lib/rules/sort-comp.js

```diff
@@ -15,6 +15,7 @@ const docsUrl = require('../util/docsUrl');
 
 const defaultConfig = {
   order: [
+    'static-variables',
```

Comment:
changing the default config is a semver-major change; it shouldn't be added here at this time.
```suggestion
```